### PR TITLE
LPS-139647 Create new LayoutPageTemplateStructure upgrade process

### PR DESCRIPTION
**Pull request by markgulacsy**
Hello @istvansajtos,
Could you give me a review on this, please?
Thanks

### CODE REVIEW

File: modules/apps/layout/layout-page-template-service/src/main/java/com/liferay/layout/page/template/internal/upgrade/v3_4_4/LayoutPageTemplateStructureUpgradeProcess.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	public LayoutPageTemplateStructureUpgradeProcess(
+		LayoutLocalService layoutLocalService) {
+
+		_layoutLocalService = layoutLocalService;
+	}
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		try (PreparedStatement preparedStatement1 = connection.prepareStatement(
+				"select plid,userId from Layout where type_ = '" +
```
Comment (istvansajtos) [inline modules/apps/layout/layout-page-template-service/src/main/java/com/liferay/layout/page/template/internal/upgrade/v3_4_4/LayoutPageTemplateStructureUpgradeProcess.java:40]: Leave a space after coma in select statement like this:
`"select plid, userId ...`

**Comment by istvansajtos** [comment]
Hi @markgulacsy ,

I see Balázs already reviewed your changes and suggested some changes. I don't have much to add except a formatting hint.

File: modules/apps/layout/layout-page-template-service/src/main/java/com/liferay/layout/page/template/internal/upgrade/v3_4_4/LayoutPageTemplateStructureUpgradeProcess.java (hunk 1 above)
Comment (markgulacsy) [reply modules/apps/layout/layout-page-template-service/src/main/java/com/liferay/layout/page/template/internal/upgrade/v3_4_4/LayoutPageTemplateStructureUpgradeProcess.java:40]: Thanks! I will fix it.

**Comment by markgulacsy** [comment]
@istvansajtos Thank you for the review!

**Pull request closed without merging**